src/styles/globals.css

```diff
@@ -78,6 +78,111 @@
   }
 }
 
+@font-face {
+  font-family: 'Sharp Sans';
+  src: url('/fonts/SharpSans-Bold.otf') format('opentype');
+  font-weight: 700;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: 'Sharp Sans';
+  src: url('/fonts/SharpSans-BoldItalic.otf') format('opentype');
+  font-weight: 700;
+  font-style: italic;
+}
+
+@font-face {
+  font-family: 'Sharp Sans';
+  src: url('/fonts/SharpSans-Book.otf') format('opentype');
+  font-weight: 400;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: 'Sharp Sans';
+  src: url('/fonts/SharpSans-BookItalic.otf') format('opentype');
+  font-weight: 400;
+  font-style: italic;
+}
+
+@font-face {
+  font-family: 'Sharp Sans';
+  src: url('/fonts/SharpSans-Extrabold.otf') format('opentype');
+  font-weight: 800;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: 'Sharp Sans';
+  src: url('/fonts/SharpSans-ExtraboldItalic.otf') format('opentype');
+  font-weight: 800;
+  font-style: italic;
+}
+
+@font-face {
+  font-family: 'Sharp Sans';
+  src: url('/fonts/SharpSans-Light.otf') format('opentype');
+  font-weight: 300;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: 'Sharp Sans';
+  src: url('/fonts/SharpSans-LightItalic.otf') format('opentype');
+  font-weight: 300;
+  font-style: italic;
+}
+
+@font-face {
+  font-family: 'Sharp Sans';
+  src: url('/fonts/SharpSans-Medium.otf') format('opentype');
+  font-weight: 500;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: 'Sharp Sans';
+  src: url('/fonts/SharpSans-MediumItalic.otf') format('opentype');
+  font-weight: 500;
+  font-style: italic;
+}
+
+@font-face {
+  font-family: 'Sharp Sans';
+  src: url('/fonts/SharpSans-Semibold.otf') format('opentype');
+  font-weight: 600;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: 'Sharp Sans';
+  src: url('/fonts/SharpSans-SemiboldItalic.otf') format('opentype');
+  font-weight: 600;
+  font-style: italic;
+}
+
+@font-face {
+  font-family: 'Sharp Sans';
+  src: url('/fonts/SharpSans-Thin.otf') format('opentype');
+  font-weight: 100;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: 'Sharp Sans';
+  src: url('/fonts/SharpSans-ThinItalic.otf') format('opentype');
+  font-weight: 100;
+  font-style: italic;
+}
+
+@font-face {
+  font-family: 'Sharp Sans Display';
+  src: url('/fonts/SharpSansDispNo1-Bold.otf') format('opentype');
+  font-weight: 700;
+  font-style: normal;
+}
```

Comment:
You've added 15 different fonts and imported all of them. So the page load time is going to be slower because you're importing fonts that are not used. Please only add the fonts that you are using and remove the others.